webhookClient/metrics.go

```diff
@@ -57,9 +60,32 @@ type Measures struct {
 	WebhookRegistrationOutcome metrics.Counter
 }
 
-//NewMeasures realizes desired metrics.
+// MeasuresIn is an uber/fx parameter with the webhook registration counter
+type MeasuresIn struct {
+	fx.In
+	WebhookRegistrationOutcome metrics.Counter `name:"webhook_registration"`
```

Comment:
I think if you add the name to the counter in `Measures`, you can embed `Measures` here? Not a big deal for one metric, but would make it easier if any metrics are added in the future.

Reply:
I think the name tagging only works if the struct has an embedded `fx.In`, but we want to keep fx separate from the `Measures` struct right?